cmd/configmap-server/main.go

```diff
@@ -39,16 +41,49 @@ func main() {
 	rootCmd.Flags().StringP("configMapName", "c", "", "name of a configmap")
 	rootCmd.Flags().StringP("configMapNamespace", "n", "", "namespace of a configmap")
 	rootCmd.Flags().StringP("port", "p", "50051", "port number to serve on")
+	rootCmd.Flags().StringP("termination-log", "t", "/dev/termination-log", "path to a container termination log file")
 	if err := rootCmd.Flags().MarkHidden("debug"); err != nil {
-		panic(err)
+		logrus.Panic(err.Error())
 	}
 
 	if err := rootCmd.Execute(); err != nil {
-		panic(err)
+		logrus.Panic(err.Error())
 	}
 }
 
 func runCmdFunc(cmd *cobra.Command, args []string) error {
+	// Immediately set up termination log
+	terminationLogPath, err := cmd.Flags().GetString("termination-log")
+	if err != nil {
+		return err
+	}
+	terminationLogFile, err := os.OpenFile(terminationLogPath, os.O_WRONLY|os.O_CREATE, 0755)
```

Comment:
only comment: we probably want to do this across all of our binaries (open the termination log file and set up the logging hooks) so it could be factored out into its own thing.

Reply:
I was thinking about making a parent cobra command with shared flags and behavior. Although, that may constrain us to a single binary.